README.md

```diff
@@ -1,6 +1,5 @@
 <div align="center">
 
-  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
   <br/>
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/218262336-ec4a1daa-da43-4f47-9b2a-c53ba269d90d.png)

- [ ] Kindly fill both of these sections to make your readme file descriptive :+1:
- [ ] Don't forget to remove the section descriptions :smile: because these are just guidelines for you :+1: